### ISSUE TITLE
Firefox display of numeric field control is confusing and undesirable

### ISSUE THREAD

**Issue by grumpyinca** [thegrumpys/odop#422]
Firefox rendering:
![image](https://user-images.githubusercontent.com/39974784/98167088-5df57c00-1e9d-11eb-81ec-42641301445f.png)

The action of the up-down arrows next to the numeric field is particularly confusing to the user.

From Alexey:
I suggest you to add something like
`input[type=number].form_control {-moz-appearance: textfield;}`
to your css.

**Comment by grumpyinca** [comment]
Update from Alexey:
If it is of interest for you, it seems possible to turn off spinners in FF. I made it this way.

1. Create file Style1.css (or the name you like :) in folder client\src\components

2. In this file add a style like
-------------------- begin -----------------
```
input[type=number] {
    -moz-appearance: textfield;
}
```
-------------------- end -------------------
(only between these begin-end)

3. Add a reference to the file into client\src\components\NameValueUnitTable.js. I have added as a last line of import:
import './Style1.css';
